pkg/resources/server.go

```diff
@@ -74,14 +137,15 @@ func newResourceServer(config *types.UserConfig, watcherMode bool, resourcePrefi
 
 	return &resourceServer{
 		resourceName: fmt.Sprintf("%s/%s", resourcePrefix, config.ResourceName),
-		socketName:   socketName,
+		endPoint:     socketName,
```

Comment:
why the name change ? also is this just the socket name and not the whole path ?

Reply:
endPoint is more suitable name

Reply:
it is not, its confusing since the grpc endpoint is the fill path

Reply:
I will change it back to `socketName`